services/mgmt/security/README.md

```diff
@@ -6,12 +6,12 @@ https://github.com/Azure/azure-rest-api-specs/blob/main/specification/security/r
 
 To get started with these generated service crates, see the [examples](https://github.com/Azure/azure-sdk-for-rust/blob/main/services/README.md#examples).
 
-The default tag is `package-preview-2023-03`.
+The default tag is `package-dotnet-sdk`.
 
 The following [tags](https://github.com/Azure/azure-sdk-for-rust/blob/main/services/tags.md) are available:
 
-- `package-preview-2023-03` has 10 operations from 2 API versions: `2023-01-01-preview`, `2023-03-01-preview`. Use crate feature `package-preview-2023-03` to enable. The operations will be in the `package_preview_2023_03` module.
-- `package-preview-2023-02` has 11 operations from 1 API versions: `2023-02-01-preview`. Use crate feature `package-preview-2023-02` to enable. The operations will be in the `package_preview_2023_02` module.
-- `package-preview-2022-11` has 4 operations from 1 API versions: `2022-11-20-preview`. Use crate feature `package-preview-2022-11` to enable. The operations will be in the `package_preview_2022_11` module.
-- `package-preview-2022-08` has 6 operations from 1 API versions: `2022-08-01-preview`. Use crate feature `package-preview-2022-08` to enable. The operations will be in the `package_preview_2022_08` module.
-- `package-preview-2022-07` has 8 operations from 1 API versions: `2022-07-01-preview`. Use crate feature `package-preview-2022-07` to enable. The operations will be in the `package_preview_2022_07` module.
\ No newline at end of file
+- `package-dotnet-sdk` has 197 operations from 20 API versions: `2015-06-01-preview`, `2017-08-01`, `2017-08-01-preview`, `2019-01-01`, `2019-01-01-preview`, `2019-08-01`, `2020-01-01`, `2020-01-01-preview`, `2021-01-15-preview`, `2021-05-01-preview`, `2021-06-01`, `2021-07-01-preview`, `2021-10-01-preview`, `2022-01-01`, `2022-01-01-preview`, `2022-03-01`, `2022-05-01`, `2022-07-01-preview`, `2022-08-01-preview`, `2023-02-01-preview`. Use crate feature `package-dotnet-sdk` to enable. The operations will be in the `package_dotnet_sdk` module.
```

Comment:
Worth noting that there are some significant changes to the security package names.  I'm presuming that this is fine - it is autogenerated and builds.

Reply:
This is a little odd, but something we can address in later updates.